lib/cocoapods/installer/pod_source_installer.rb

```diff
@@ -186,6 +153,13 @@ def root
         sandbox.pod_dir(root_spec.name)
       end
 
+      # @return [Boolean] whether the source has already been downloaded prior
+      #         to the installation process.
+      #
+      def downloaded?
```

Comment:
are the methods here and below still used in this file now that they've been added to the new class `PodSourceDownloader`?

Reply:
Yes, it's still used in the `download_source` method, which calls into the `PodSourceDownloader`